backend/tracim_backend/lib/core/user.py

```diff
@@ -932,7 +932,9 @@ def _check_user_auth_validity(self, user:User) -> None:
             raise UserAuthTypeDisabled('user {} auth type {} is disabled'.format(user.email, user.auth_type.value))
 
     def _user_can_authenticate(self, user: User) -> bool:
-        return user.auth_type and user.auth_type in self._config.AUTH_TYPES
+        valid_auth_types = list(self._config.AUTH_TYPES)
+        valid_auth_types.append(AuthType.UNKNOWN)
```

Comment:
please add a comment and a link to the issue

Reply:
done.